spec/integration/inputs/kafka_spec.rb

```diff
@@ -91,6 +99,55 @@
         expect(event.get("[@metadata][kafka][timestamp]")).to be >= start
       end
     end
+
+    it "should show the right topic and group name in and kafka headers decorated kafka section" do
+      header = org.apache.kafka.common.header.internals.RecordHeader.new("name", "John".to_java_bytes)
```

Comment:
Since we are validating that UTF-8 can get through, we should include sequences from a variety of higher-order UTF-8 characters -- not just US-ASCII.